scripts/convert_csv_to_json.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+import pandas as pd
+import sys
+import json
+
+
+def generate_json_files(version):
+
+    source_directory = './source-data/'
+    output_directory = './data/' + version + '/'
+
+    for source_file_name in os.listdir(source_directory):
+        if os.path.isfile(os.path.join(source_directory, source_file_name)):
+
+            source_file = open(os.path.join(source_directory, source_file_name), 'r')
+
+            output_file_name = source_file_name.replace('.csv', '.json')
+            output_file_location = os.path.join(output_directory, output_file_name)
+
+            csv_data = pd.DataFrame(
+                pd.read_csv(source_file,
+                            sep=",",
+                            header=0,
+                            index_col=False))
+
+            try:
+                # This approach will exclude null values
+                with open(output_file_location, 'w') as f:
+                    json.dump([row.dropna().to_dict() for index, row in csv_data.iterrows()], f, indent=3)
```

Comment:
So if I had England with no suggest for Welsh I would end up with {'"en-gb": "England"} is that what is expected in runner?

Reply:
Runner doesn't use this, only the front-end design-system component. But yes, that's the requirement hence this implementation does not output null value keys.

Reply:
Doesn't that mean there would be missing entries? In runner if there is no translation for something it falls back to the English text.

Reply:
That's correct. Its completely up to the business what values they want presenting in each language (and for them to provide those translations in the source file). However it does look like for the countries-of-birth there is a direct 1:1 (English:Welsh. That may or may not be the case for other lists. This allows for there to be 'missing' entries in a given language if the business so wish. I don't know why they'd want that, but its up to them to provide the files as they require.